src/main/java/uk/gov/hmcts/darts/task/service/impl/LockServiceImpl.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.darts.task.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import net.javacrumbs.shedlock.core.DefaultLockingTaskExecutor;
+import net.javacrumbs.shedlock.core.LockProvider;
+import net.javacrumbs.shedlock.core.LockingTaskExecutor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.common.entity.AutomatedTaskEntity;
+import uk.gov.hmcts.darts.common.helper.CurrentTimeHelper;
+import uk.gov.hmcts.darts.common.repository.AutomatedTaskRepository;
+import uk.gov.hmcts.darts.task.service.LockService;
+
+import java.sql.Timestamp;
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class LockServiceImpl implements LockService {
+
+    private static final int DEFAULT_LOCK_AT_MOST_SECONDS = 600;
+    private static final int DEFAULT_LOCK_AT_LEAST_SECONDS = 20;
```

Comment:
Consider declaring these as `Duration` to avoid the need for the descriptive name and convertion in the getters:

```suggestion
    private static final Duration DEFAULT_LOCK_AT_MOST = Duration.ofSeconds(600);
    private static final Duration DEFAULT_LOCK_AT_LEAST = Duration.ofSeconds(20);
```